platforms/windows/src/node.rs

```diff
@@ -312,6 +317,92 @@ impl NodeWrapper<'_> {
                 .unwrap();
         }
 
+        if self.0.is_busy() {
+            properties.write_bool_property("busy", true).unwrap();
+        }
+
+        if let Some(colindextext) = self.0.column_index_text() {
+            properties
+                .write_property("colindextext", colindextext)
+                .unwrap();
+        }
+
+        if let Some(current) = self.0.aria_current() {
+            if current != AriaCurrent::False {
+                fn val(c: AriaCurrent) -> &'static str {
+                    match c {
+                        AriaCurrent::True => "true",
+                        AriaCurrent::Page => "page",
+                        AriaCurrent::Step => "step",
+                        AriaCurrent::Location => "location",
+                        AriaCurrent::Date => "date",
+                        AriaCurrent::Time => "time",
+                        AriaCurrent::False => unreachable!(),
+                    }
+                }
+
+                properties.write_property("current", val(current)).unwrap();
+            }
+        }
+
+        if let Some(has_popup) = self.0.has_popup() {
+            fn val(h: HasPopup) -> &'static str {
+                match h {
+                    HasPopup::Menu => "menu",
+                    HasPopup::Listbox => "listbox",
+                    HasPopup::Tree => "tree",
+                    HasPopup::Grid => "grid",
+                    HasPopup::Dialog => "dialog",
+                }
+            }
+
+            properties
+                .write_property("haspopup", val(has_popup))
+                .unwrap();
+        }
+
+        if let Some(level) = self.0.level() {
+            properties
+                .write_property("level", &level.to_string())
+                .unwrap();
+        }
+
+        if self.0.is_multiline() {
+            properties.write_bool_property("multiline", true).unwrap();
+        }
+
+        if let Some(posinset) = self.0.position_in_set() {
```

Comment:
As said before, we already expose this property (`UIA_PositionInSetPropertyId`). Still if you think it should appear here, please use `NodeWrapper::position_in_set` because it takes care of adding one to `Node::position_in_set` which is zero-based.